### ISSUE TITLE
Cannot load Qwen-Image-Edit-2509

### ISSUE THREAD

**Issue by mccatec** [modelscope/DiffSynth-Studio#959]
I modified example code as follows, since my env has no internet connection. Error occurs while loading model as:
...
Using qwen_image_vae from ['/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/vae/diffusion_pytorch_model.safetensors'].
No qwen_image_blockwise_controlnet models available.

my code:
from diffsynth.pipelines.qwen_image import QwenImagePipeline, ModelConfig
from PIL import Image
import torch

pipe = QwenImagePipeline.from_pretrained(
    torch_dtype=torch.bfloat16,
    device="cuda",
    model_configs=[
        ModelConfig(path=[
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/transformer/diffusion_pytorch_model-00001-of-00005.safetensors",
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/transformer/diffusion_pytorch_model-00002-of-00005.safetensors",
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/transformer/diffusion_pytorch_model-00003-of-00005.safetensors",
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/transformer/diffusion_pytorch_model-00004-of-00005.safetensors",
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/transformer/diffusion_pytorch_model-00005-of-00005.safetensors",
        ], skip_download=True),
        ModelConfig(path=[
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/text_encoder/model-00001-of-00004.safetensors",
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/text_encoder/model-00002-of-00004.safetensors",
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/text_encoder/model-00003-of-00004.safetensors",
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/text_encoder/model-00004-of-00004.safetensors",
        ], skip_download=True),
        ModelConfig(path=[
            "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/vae/diffusion_pytorch_model.safetensors"
        ], skip_download=True),
    ],
    processor_config=ModelConfig(path=[
        "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/processor/added_tokens.json",
        "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/processor/chat_template.jinja",
        "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/processor/preprocessor_config.json",
        "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/processor/special_tokens_map.json",
        "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/processor/tokenizer_config.json",
        "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/processor/tokenizer.json",
        "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/processor/video_preprocessor_config.json",
        "/home/xx/models/huggingface.co/Qwen/Qwen-Image-Edit-2509/processor/vocab.json",
    ], skip_download=True)
)

image_1 = pipe(prompt="一位少女", seed=0, num_inference_steps=40, height=1328, width=1024)
image_1.save("image1.jpg")

image_2 = pipe(prompt="一位老人", seed=0, num_inference_steps=40, height=1328, width=1024)
image_2.save("image2.jpg")

prompt = "生成这两个人的合影"
edit_image = [Image.open("image1.jpg"), Image.open("image2.jpg")]
image_3 = pipe(prompt, edit_image=edit_image, seed=1, num_inference_steps=40, height=1328, width=1024, edit_image_auto_resize=True)
image_3.save("image3.jpg")

**Comment by pd162** [comment]
Have you solved this problem? The same error.

**Comment by mkamranr** [comment]
Facing the exact same issue, works fine on internet. Loading weights by putting paths still tries to connect to internet at modelscope.cn
